docker-hub/orgs.md

```diff
@@ -163,7 +163,7 @@ Use the following steps to invite a member to your organization via Docker ID or
 
 1. Go to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
 2. In the **Members** tab, select **Invite Member**.
-3. Select **Docker ID/Email**.
+3. Select **Emails Or Docker IDs**.
```

Comment:
Should lower case "or"?

Reply:
Thanks, @jerae-duffin. It should lowercase, but the docs currently match the UI wording where it's uppercase. There's an issue with the design system that must be fixed first. It's tracked in CSP-116 and it doesn't appear that it'll be fixed before release.

Reply:
Thanks Craig! LGTM!